backend/algorithms/exampleUnselectCourse.json

```diff
@@ -2,7 +2,7 @@
     "userData": {
```

Comment:
Looks like `exampleUnselectCourse.json` just contains one example user. There already exists `exampleUsers.json` in the same directory as: `/backend/algorithmns/exampleUsers.json`
Ideally, this user would get added to that file instead so that this file can be deleted.